CrewChiefV4/CrewChief.cs

```diff
@@ -427,6 +427,12 @@ public Boolean Run(String filenameToRun, int interval, Boolean dumpToFile)
                             isGameProcessRunning = Utilities.IsGameRunning(gameDefinition.processName, gameDefinition.alternativeProcessNames);
                         }
 
+                        if (mapped && !isGameProcessRunning)
```

Comment:
for Pcars UDP data reading, mapped will be true and isGameProcessRunning will be false, so this will always call Disconnect then set mapped to false, which is wrong (I think). Should this check be something like:
if (mapped && gameDefinition.processName != null && !isGameProcessRunning) {....}